.github/workflows/lint-and-test.yml

```diff
@@ -56,6 +56,13 @@ jobs:
         run: |
           kubectl apply -f charts/argo-cd/crds
 
+      # - name: Deploy latest ArgoCD
+      #   if: |
+      #     contains(steps.list-changed.outputs.changed_charts, 'argocd-applicationset')
+      #   run: |
+      #     kubectl create namespace argocd
+      #     kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml
+
```

Comment:
Leaving this commented for now ? Could be useful in the future ?

Reply:
I think we should remove this. Otherwise I'd like to use the Chart variant we use in "our" helm space here :-)

Reply:
makes sense.